packages/client/package.json

```diff
@@ -35,9 +35,9 @@
     "IE 10"
   ],
   "devDependencies": {
-    "@babel/core": "^7.5.5",
-    "@babel/polyfill": "^7.4.4",
-    "@babel/preset-env": "^7.5.5",
+    "@babel/core": "^7.6.4",
+    "@babel/polyfill": "^7.6.0",
```

Comment:
`@babel/polyfill` is deprecated: https://babeljs.io/docs/en/babel-polyfill

Reply:
Removed since it wasn't used anyway.